src/reference/config/catalogs/catalog-filesystem.md

```diff
@@ -0,0 +1,65 @@
+---
+shortTitle: filesystem
+description: Zilla runtime filesystem catalog
+category:
+  - Catalog
+---
+
+# filesystem Catalog
+
+::: important Feature is in Incubator
+Read how to [enable incubator features](../../../how-tos/deploy-operate.md#enable-incubator-features). Star and watch the [Zilla repo](https://github.com/aklivity/zilla/releases) for new releases!
+:::
+
+Zilla runtime filesystem catalog
+
+```yaml {2}
+catalogs:
+  catalog0:
+    type: filesystem
+    options:
+      subjects:
+        petstore:
+          url: schema.yaml
+```
+
+## Summary
+
+Defines a catalog with schemas, AsyncAPI/OpenAPI definitions or proto files pulled from the filesystem relative `zilla.yaml` to enforce validation, create APIs or gRPC services.
+
+## Configuration
+
+:::: note Properties
+
+- [options](#options)
+  - [options.subjects\*](#options-subjects)
+    - [subjects.url\*](#subjects-url)
+    - [subjects.version](#subjects-version)
+
+::: right
+\* required
+:::
+
+::::
+
+### options
+
+> `object`
+
+#### options.subjects\*
+
+> `object` as map of named objects
+
+Unique identifier for artifact categorization in the catalog.
+
+##### subjects.url\*
```

Comment:
If this is a local or remote location then `url` is probably fine, but if we are going to limit it to the host filesystem then this should be `path` or `relative-path`?

Reply:
Right now readURL supports remote location as well. @jfallows , what do you think?

Reply:
This should be `path` for `filesystem` catalog.

Internally, `readURL` is scheduled for removal.